inspectit-gepard-agent/src/main/java/rocks/inspectit/gepard/agent/internal/properties/PropertiesResolver.java

```diff
@@ -90,4 +97,41 @@ public static Duration getPollingInterval() {
         ? Duration.parse(pollingIntervalEnvProperty)
         : Duration.ofSeconds(10);
   }
+
+  /**
+   * Get the attributes which should be sent to the configuration server. When an attribute is
+   * configured via both system properties and environmental properties, the system property will be
+   * used.
+   *
+   * @return the attributes to be sent to the configuration server
+   */
+  public static Map<String, String> getAttributes() {
+
+    Map<String, String> attributes = new HashMap<>(getAttributesFromEnv());
+
+    // Override with system properties where available
+    attributes.putAll(getAttributesFromSystemProperties());
+
+    return attributes;
+  }
+
+  private static Map<String, String> getAttributesFromEnv() {
```

Comment:
Could you still add java docs to the private methods, since the stream operations are not too simple.